### PR TITLE
Changed skip_pathway_analysis to run_pathway_analysis (10)

### LINKED ISSUE #168: Rename the parameter --skip_pathway_analysis to --run_pathway_analysis and have the default set to 'false'
### Description of feature

We should rename the parameter `--skip_pathway_analysis` to  `--run_pathway_analysis` and have the default value set to `false`, thus that pathway analysis is NOT run as default. 

Background: We have had some issues in the past on multiple projects where we did not offer pathway analysis on the offer but were running and shipping the results as the default of the workflow is currently set to run it. 
The above renaming and setting of the default to `false` will make it much easier and less error-prone to actively switch-on pathway analysis when we in-deed offer the service, and set it to false when we do not offer it, thus allowing a clearer connection between data analysis and Project Management. In the near future we can think of a switch to `true` for example when we integrate the costs for pathway analysis in the costs for secondary Bx analysis in our current price/cost model for the Offer manager.

### PR DESCRIPTION
**Pull request by WackerO**
Parameter is for clarity's sake now called run_pathway_analysis, default false, as BioPM has to add pathway analysis specifically to offers as well -->closes #168 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnadeseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/rnadeseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -291,9 +291,9 @@ This is ignored if --use_vst is set to false. If using the vst transformation, c
 
 Integer indicating to how many decimals to round the DE results (default: no rounding).
 
-### `--skip_pathway_analysis`
+### `--run_pathway_analysis`
 
-Set this flag to 'true' to skip pathway analysis and only run differential gene expression and report generation.
+Set this flag to 'true' to run pathway analysis, otherwise, this step will be skipped.
```
Comment (ggabernet) [inline docs/usage.md:296]:
```suggestion
Set this flag to run pathway analysis, otherwise, this step will be skipped.
```

File: nextflow.config [hunk 2]
```diff
@@ -69,6 +69,10 @@ params {
     schema_ignore_params  = null
 }
 
+// Container slug. Stable releases should specify release tag!
+// Developmental code should specify :dev
+process.container = 'qbicpipelines/rnadeseq:dev'
+
```
Comment (ggabernet) [inline nextflow.config:75]:
```suggestion
```

Comment (ggabernet) [reply nextflow.config:75]: Is there a reason why this was added again? We're in DSL2 so the container is specified within the module, or?

**Review by ggabernet — APPROVED** [review]
Just some comments, otherwise LGTM

File: nextflow.config (hunk 2 above)
Comment (WackerO) [reply nextflow.config:75]: Oop, you're right of course!

**Pull request merged**